st2common/st2common/validators/api/reactor.py

```diff
@@ -70,4 +74,12 @@ def validate_trigger_parameters(trigger_type_ref, parameters):
     cleaned = util_schema.validate(instance=parameters, schema=parameters_schema,
                                    cls=util_schema.CustomValidator, use_default=True,
                                    allow_default_none=True)
+
+    # Additional validation fron CronTimer
+    if trigger_type_ref == ResourceReference.to_string_reference(SYSTEM_PACK_NAME, 'st2.CronTimer'):
```

Comment:
I'm not actually sure what you mean.

Right now this function is the place where additional, trigger-specific parameter validation is performed (hence it's called `validate_trigger_parameters`).